src/Dropdown.vue

```diff
@@ -1,27 +1,25 @@
 <template>
   <li v-if="isLi" ref="dropdown" :class="classes">
     <slot name="button">
-      <a class="dropdown-toggle" role="button" :class="{disabled: disabled}" @keyup.esc="show = false">
+      <a class="dropdown-toggle" role="button" :class="{disabled: disabled}" @keyup.esc="hideDropdownMenu()">
         {{ text }}
-        <span class="caret"></span>
       </a>
     </slot>
-    <slot name="dropdown-menu">
-      <ul class="dropdown-menu">
+    <slot name="dropdown-menu" :class="menuClasses">
+      <ul class="dropdown-menu" :class="menuClasses">
         <slot></slot>
       </ul>
     </slot>
   </li>
   <div v-else ref="dropdown" :class="classes">
     <slot name="before"></slot>
```

Comment:
Currently the `.btn` class has `padding: .375rem .75rem;`, which makes Segmented buttons quite fat. maybe add some conditional rendering for the dropdown button to slim down when it has contents within the **before slot**?